src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -71,80 +24,88 @@ public override void _Ready()
         stage = (MicrobeStage)GetParent();
     }
 
-    public override void _UnhandledInput(InputEvent @event)
-    {
-        if (inputs.CheckInput(@event))
-        {
-            GetTree().SetInputAsHandled();
-        }
-    }
-
     public override void _Notification(int focus)
     {
         // If the window goes out of focus, we don't receive the key released events
         // We reset our held down keys if the player tabs out while pressing a key
         if (focus == MainLoop.NotificationWmFocusOut)
         {
-            inputs.FocusLost();
+            InputManager.FocusLost();
         }
     }
```

Comment:
Why is this sent to the input manager? Why doesn't input manager implement _Nofitication?